app/src/main/java/ca/bc/gov/bchealth/ui/login/BcscAuthViewModel.kt

```diff
@@ -121,6 +121,13 @@ class BcscAuthViewModel @Inject constructor(
             }
             val isLoggedSuccess = bcscAuthRepo.processAuthResponse(data)
             if (isLoggedSuccess) {
+                // clear medication pref
+                cacheRepository.updateProtectiveWordAdded(false)
+                clearStorageRepository.clearMedicationPreferences()
+
+                // cancel work manager
+                WorkManager.getInstance(context).cancelUniqueWork(BACKGROUND_AUTH_RECORD_FETCH_WORK_NAME)
```

Comment:
post logout work should stop immediately. With above shift that was not happening. We need to revert this change.